kwsStyle.cxx

```diff
@@ -402,19 +402,19 @@ int main(int argc, const char **argv)
       return 0;
       }
     file.seekg(0,std::ios::end);
-    unsigned long fileSize = file.tellg();
+    unsigned long fileSize = static_cast<unsigned long>(file.tellg());
```

Comment:
Also change variable type from `ulong` to `size_type` or whatever `tellg` returns.